source/ed/build_helper.cpp

```diff
@@ -149,14 +149,14 @@ nt::VehicleJourney* VJ::make() {
     const auto uri_str =
         _uri.empty() ? "vj:" + line_name + ":" + std::to_string(pt_data.vehicle_journeys.size()) : _uri;
     if (is_frequency) {
-        auto* fvj = mvj->create_frequency_vj(uri_str, vj_type, _vp, route, sts, pt_data);
+        auto* fvj = mvj->create_frequency_vj(uri_str, uri_str, vj_type, _vp, route, sts, pt_data);
```

Comment:
Twice the same parameter?

Reply:
Added a commit to differentiate this in tests... I know why I didn't want to do that: it's fat ;)